_lifeofinternatpuppet/post_7.md

```diff
@@ -94,13 +120,6 @@ I couldn't write the blog update that week as I wasn't well that day, so sorry f
 
 I attended the Friday Wrap Up as usual, had a 1:1 with Caoimhe as I had some questions, don't remember exactly what they were, sorry again :)
 
-I was invited to a zoom call by Ciaran and David Schmitt, where they were trying to debug issues on Crontab and its configuration set up.
-
-The error/issue with a solution in Ciaran's own words :-
-
-"Error/Issue: cron is not executing the configured scripts as expected. Permission errors on crontab file being reported to syslog. Additional issue of “can’t fork” errors manifesting this morning
-solution suggested was wrap the command in bash -c to ensure it’s correctly invoked, but the problem still persisted"
-
-During that demo session with Ciaran and David Schmitt, I also asked Ciaran if he had a sync up with Paula or Daiana from yesterday's 1:1 to see if he knows what I did with Paula and Diana/what are the plans for the upcoming week. Ciaran said 'due to other commitments he hasn't had a chance I think, don't remember exactly what he said during the session.
+I was invited to a zoom call by Ciaran and David Schmitt, where they were trying to debug issues on Crontab and its configuration set up. During that demo session with Ciaran and David Schmitt, I also asked Ciaran if he had a sync up with Paula or Daiana from yesterday's 1:1 to see if he knows what I did with Paula and Diana/what are the plans for the upcoming week. Ciaran said 'due to other commitments he hasn't had a chance I think, don't remember exactly what he said during the session.
```

Comment:
Use another approach here:
 _don't remember exactly what he said during the session._